--- a/csrc/jit/device_runtime.hpp
+++ b/csrc/jit/device_runtime.hpp
@@ -93,10 +93,10 @@
                 return "100";
             return support_arch_family ? "100f" : "100a";
         }
-        if (major == 12 and minor != 1) {
+        if (major == 12) {
             if (number_only)
                 return "120";
-            return support_arch_family ? "120f" : "120a";
+            return minor == 1 ? "121a" : "120a";
         }
         return std::to_string(major * 10 + minor) + (number_only ? "" : "a");
     }
